src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs

```diff
@@ -6,104 +6,96 @@
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Hosting;
 using System.Runtime.InteropServices;
-using System.Text;
-using Vanara.PInvoke;
 using Windows.Win32;
-using Windows.Win32.System.Com;
+using Windows.Win32.Foundation;
 using Windows.Win32.Graphics.Direct3D;
 using Windows.Win32.Graphics.Direct3D11;
 using Windows.Win32.Graphics.DirectComposition;
 using Windows.Win32.Graphics.Dwm;
 using Windows.Win32.Graphics.Dxgi;
+using Windows.Win32.System.Com;
+using Windows.Win32.UI.Shell;
 using Windows.Win32.UI.WindowsAndMessaging;
 using WinRT;
-using static Vanara.PInvoke.ShlwApi;
-using static Vanara.PInvoke.User32;
-using System.Runtime.InteropServices.Marshalling;
-
 
 #pragma warning disable CS8305 // Type is for evaluation purposes only and is subject to change or removal in future updates.
```

Comment:
What is this?

Reply:
This is to supress that warning that shows up because ContentExternalOutput has ExperimentalAttribute on it.